src/Objects/Filesystem/File.php

```
@@ -4,27 +4,12 @@
 
 use EncoreDigitalGroup\StdLib\Exceptions\FilesystemExceptions\FileNotFoundException;
 
-/** @api */
 class File
 {
-    /**
-     * @deprecated use File::content() instead.
-     *
-     * @codeCoverageIgnore
-     */
-    public static function getContent(string $path): string
-    {
-        return self::content($path);
-    }
-
-    /**
-     * @experimental
-     *
-     * @codeCoverageIgnore
-     */
+    /** @codeCoverageIgnore */
     public static function content(string $path): string
     {
-        $file = file_get_contents($path);
+        $file = @file_get_contents($path);
```

hiding all errors I see :)

---

Yep! I want to throw a specific error for this lib.